src/huts/repositorySpec.js

```diff
@@ -45,17 +45,17 @@ describe("huts repository" , function() {
             expect(actualError).not.toBe(null);
         });
 
-        it("explodes if the database explodes while creating the huts table, if the table does not already exist", async function() {
-            mockFailOnQuery(db, [
-                "SELECT 'public.huts'::regclass", 
-                "CREATE TABLE huts(id uuid UNIQUE PRIMARY KEY, data json NOT NULL)"
-            ]);
-            
+        fit("explodes if the database explodes while creating the huts table, if the table does not already exist", async function() {
```

Comment:
`fit` should probably not be left in here :smile:

When committing in the future, it would make sense to fix it back to `it`, add and commit, then change it to `fit` again. I don't need to know that you've been running this one test alone :smile: